### DISCUSSION TITLE
SUB_QUALITY Env Options

### DISCUSSION THREAD

**Discussion by dimmanramone** [mrlt8/docker-wyze-bridge#771] [Q&A]
Hi,

Does anyone know how to use the new SUB_QUALITY env option?

I cannot find anything in the documentation

**Comment by JA16122000** [comment]
#755

**Comment by mrlt8** [comment]
`SUB_QUALITY` should take the same format as `QUALITY` so something like: `SUB_QUALITY=SD60`
